dockstore-integration-testing/src/test/java/io/dockstore/common/CommonTestUtilities.java

```diff
@@ -357,9 +357,9 @@ public static void restartElasticsearch() throws Exception {
             Container container = elasticsearch.get();
             try {
                 docker.restartContainer(container.id());
-                // Wait 15 seconds for elasticsearch to become ready
+                // Wait 25 seconds for elasticsearch to become ready
                 // TODO: Replace with better wait
-                Thread.sleep(15000);
+                Thread.sleep(25000);
```

Comment:
1. Does the new ES take longer to start up?
2. A more reliable way would be to run a command against the container, but probably not worth the effort. I think this is only called for a few tests. But if it's called by every test, it's going to significantly increase our build time (100 tests * 10 seconds) = 16 minutes (although we run in parallel).

Reply:
Yup, the new ES wasn't starting within 15 seconds